flintrock/config.yaml.template

```diff
@@ -1,6 +1,7 @@
 services:
   spark:
     version: 1.6.0
+    # preferred-mirror: # optional; default to 'https://s3.amazonaws.com/spark-related-packages/${file}'
```

Comment:
If you're talking about the config template, I think we just need a `{version}` variable. It seems unnecessary to say `spark_...` inside the Spark config.

As for the Hadoop distribution, we currently don't support specifying that, unless you are also intending to tackle #88.